katalog/forecastle/rbac.yml

```diff
@@ -12,14 +16,12 @@ metadata:
   labels:
     app: forecastle
 rules:
-  - apiGroups:
-      - extensions
-      - networking.k8s.io
-    resources:
-      - ingresses
-    verbs:
-      - get
-      - list
+  - apiGroups: ["", "route.openshift.io", "extensions"]
```

Comment:
I don't think we need RBAC for OpenShift Routes or Forecastle CRD, anyway they do no harm. :D